.idea/deploymentTargetDropDown.xml

```diff
@@ -7,11 +7,11 @@
         <deviceKey>
           <Key>
             <type value="VIRTUAL_DEVICE_PATH" />
-            <value value="$USER_HOME$/.android/avd/Pixel_4_API_30.avd" />
+            <value value="C:\Users\spor6\.android\avd\Pixel_2_API_30_1.avd" />
```

Comment:
it seems like there is a path hardcoding, I think it would be better to move it back to USER_HOME